packages/vue/src/api.ts

```diff
@@ -0,0 +1,433 @@
+   */
+  public async getIDPAccessToken(): Promise<string> {
+    return this._client.getIDPAccessToken();
+  }
+
+  /**
+   * This method refreshes the access token.
+   *
+   * @return {BasicUserInfo} - A Promise that resolves with an object containing
+   * information about the refreshed access token.
+   */
+  public async refreshAccessToken(): Promise<BasicUserInfo> {
+    return this._client.refreshAccessToken();
+  }
+
+  /**
+   * This method specifies if the user is authenticated or not.
+   *
+   * @return {Promise<boolean>} - A Promise that resolves with `true` if teh user is authenticated.
+   */
+  public async isAuthenticated(): Promise<boolean> {
+    return this._client.isAuthenticated();
+  }
+
+  /**
+   * This method specifies if the session is active or not.
+   *
+   * @return {Promise<boolean>} - A Promise that resolves with `true` if there is an active session.
+   */
+  public async isSessionActive(): Promise<boolean> {
+    return this._client.isSessionActive();
+  }
+
+  /**
+   * This method enables callback functions attached to the http client.
+   *
+   * @return {Promise<boolean>} - A promise that resolves with True.
+   *
+   */
+  public async enableHttpHandler(): Promise<boolean> {
+    return this._client.enableHttpHandler();
+  }
+
+  /**
+   * This method disables callback functions attached to the http client.
+   *
+   * @return {Promise<boolean>} - A promise that resolves with True.
+   */
+  public async disableHttpHandler(): Promise<boolean> {
+    return this._client.disableHttpHandler();
+  }
+
+  /**
+   * This method updates the configuration that was passed into the constructor when instantiating this class.
+   *
+   * @param {Partial<AuthClientConfig<T>>} config - A config object to update the SDK configurations with.
+   */
+  public async updateConfig(config: Partial<AuthClientConfig<Config>>): Promise<void> {
+    return this._client.updateConfig(config);
+  }
+
+  /**
+   * This method attaches a callback function to an event hook that fires the callback when the event happens.
+   *
+   * @param {Hooks.CustomGrant} hook - The name of the hook.
+   * @param {(response?: any) => void} callback - The callback function.
+   * @param {string} id (optional) - The id of the hook. This is used when multiple custom grants are used.
+   *
+   */
+  public on(hook: Hooks.CustomGrant, callback: (response?: any) => void, id: string): Promise<void>;
+  public on(hook: Exclude<Hooks, Hooks.CustomGrant>, callback: (response?: any) => void): Promise<void>;
+  public on(hook: Hooks, callback: (response?: any) => void, id?: string): Promise<void> {
+    if (hook === Hooks.CustomGrant) {
+      return this._client.on(hook, callback, id);
+    }
+
+    return this._client.on(hook, callback);
+  }
+
+  /**
+   * This method allows you to sign in silently.
+   * First, this method sends a prompt none request to see if there is an active user session in the identity server.
```

Comment:
These doc comments are visible to end users who try to integrate auth with this SDK. Since we need to keep interoperability even though we go with asgardeo branding, better if we keep comments vendor agnostic.

```suggestion
   * First, this method sends a prompt none request to see if there is an active user session in the identity provider.
```

@brionmario WDYT?